unknown_type/[M]739.cpp

```diff
@@ -0,0 +1,29 @@
+// Problem: 739. Daily Temperatures - Medium
+// Link: https://leetcode.com/problems/daily-temperatures/
+
+/* Nhận xét:
+- Anh đã lọc rồi nhưng có vẻ leetcode đánh tag có vấn đề nên ra bài này. Tiện tay thì anh cứ làm luôn, sry chú.
+- Bài này có vẻ khá nổi tiếng, lưu một chuỗi giảm dần. Anh giải được nhưng nói thật là cũng chưa biết giải thích thế nào cho clear. Chú giải thích hộ anh nhé?
+*/
+
+class Solution {
+public:
+    vector<int> dailyTemperatures(vector<int>& t) {
+        int n = t.size();
+        vector<int> res(n, 0);
+        stack<pair<int, int>> st;
+        for (int i = 0; i < n; ++i) {
```

Comment:
Bài này chú for ngược từ cuối về đầu sẽ dễ code hơn, có thể loại đi nhiều biến thừa. Giải thích thì chắc phải call mới nói rõ được. Đây là dạng bài rất hay ra trong pv, chú cứ luyện được thì tốt.